src/main/java/org/mskcc/cbio/oncokb/web/rest/CronJobController.java

```diff
@@ -274,6 +231,41 @@ public void analyzeUserUsage() throws IOException {
         log.info("User usage analysis completed!");
     }
 
+    /**
+     * {@code GET /wrap-token-stats}: Wrap token stats
+     *
+     * @throws IOException
+     */
+    @GetMapping(path="move-token-stats-to-s3")
+    public void moveTokenStatsToS3() throws IOException {
+        log.info("Started the cronjob to move token stats to s3.");
+
+        Instant tokenUsageDateBefore = ZonedDateTime.now(ZoneId.of("America/New_York"))
```

Comment:
We don't do NYC time, but UTC.

Reply:
I don't think it's an issue, the time is all relevant. All token states are recorded in UTC, the cronjob is ran on UTC. Unless I'm missing anything?

Reply:
Ah sorry, I forgot cronjob is run automatically and will be the same time every day anyway. I'll change that back now.

Reply:
Should be good now^